### PR TITLE
Removing references to deprecated code

### PR DESCRIPTION
**Pull request by CatGarab**
Removing everything I could find related to:
- RT
- Telemetry
- C2
- EPS API

### CODE REVIEW

File: docs/sdk-docs/sdk-cheatsheet.rst [hunk 1]
```diff
@@ -63,27 +60,17 @@ Kubos needs to know which target you intend to build for so it can
...
-| Pumpkin    | kubos-linux-pumpkin-mbm2-gcc | Pumpkin Motherboard Module 2                      | KubOS Linux  |
-+------------+------------------------------+---------------------------------------------------+--------------+  
-| Beaglebone | kubos-linux-beaglebone-gcc   | Beaglebone Black, Rev. C                          | KubOS Linux  |
-+------------+------------------------------+---------------------------------------------------+--------------+
-|            | legacy-linux-beaglebone-gcc  | Beaglebone Black running a non-Kubos Linux distro | Debian       |
-+------------+------------------------------+---------------------------------------------------+--------------+
-| (Vagrant)  | x86-linux-native             | Native target for the Kubos Vagrant image         | KubOS Linux  |
-+------------+------------------------------+---------------------------------------------------+--------------+
++------------+------------------------------+---------------------------------------------------+
+| MCU Family | Kubos Target                 | Description                                       |
```
Comment (plauche) [inline docs/sdk-docs/sdk-cheatsheet.rst]: I don't know if `MCU Family` is accurate anymore. Maybe `OBC Manufacturer` or `Vendor`?

File: docs/sdk-docs/sdk-project-config.rst [hunk 2]
```diff
@@ -1007,26 +986,9 @@ These are the configuration options which are most likely to be changed for a pr
...
-    - "cmd-control-daemon": "kubos/cmd-control-daemon"
-    
-        - "csp": "kubos/libcsp"
-        - "command-and-control": "kubos/command-and-control"
-        - "ipc": "kubos/ipc"
-        - "tinycbor": "kubos/tinycbor"
-        - "kubos-core": "kubos/kubos-core"
+    Available dependency name/value pairs (hierarchy denotes included dependencies. Italics denotes Yotta targetDependencies):
         
     - "cmsis-core": "kubos/cmsis-core"
```
Comment (plauche) [inline docs/sdk-docs/sdk-project-config.rst]: Can remove

File: docs/sdk-docs/sdk-project-config.rst [hunk 3]
```diff
@@ -1044,99 +1006,23 @@ These are the configuration options which are most likely to be changed for a pr
                 
                     - "cmsis-core": "kubos/cmsis-core"
```
Comment (plauche) [inline docs/sdk-docs/sdk-project-config.rst]: Can remove

File: docs/sdk-docs/sdk-project-config.rst [hunk 4]
```diff
@@ -1044,99 +1006,23 @@ These are the configuration options which are most likely to be changed for a pr
                 
                     - "cmsis-core": "kubos/cmsis-core"
                     
-    - "command-and-control": "kubos/command-and-control"
     - "csp": "kubos/libcsp"
     
         - `"freertos": "kubos/freertos"`
```
Comment (plauche) [inline docs/sdk-docs/sdk-project-config.rst]: Can remove

File: docs/sdk-docs/sdk-project-config.rst [hunk 5]
```diff
@@ -1044,99 +1006,23 @@ These are the configuration options which are most likely to be changed for a pr
...
-    
-        - `"cmsis-core": "kubos/cmsis-core"`
-        - `"freertos-config-stm32f4": "kubos/freertos-config-stm32f4"`
-        - `"freertos-config-msp430f5529": "kubos/freertos-config-msp430f5529"`
-        
     - "ipc": "kubos/ipc"
     
         - "csp": "kubos/libcsp"
         - "tinycbor": "kubos/tinycbor"
         - `"kubos-rt": "kubos/kubos-rt"`
```
Comment (plauche) [inline docs/sdk-docs/sdk-project-config.rst]: Can remove

File: docs/sdk-docs/sdk-reference.rst [hunk 6]
```diff
@@ -117,7 +117,7 @@ Options
 ~~~~~~~
 
 -  ``--linux``, ``-l`` Create the new project as a linux application for
-   KubOS Linux
+   Kubos Linux
 -  ``--rt``, ``-r`` Create the new project as a KubOS RT project
```
Comment (plauche) [inline docs/sdk-docs/sdk-reference.rst]: Can remove

File: docs/sdk-docs/sdk-cheatsheet.rst (hunk 1 above)
Comment (CatGarab) [reply docs/sdk-docs/sdk-cheatsheet.rst]: Ooh, I like `Vendor`, that's much better

**Pull request merged**